test/parameter_test.py

```diff
@@ -678,6 +702,14 @@ def complete(self):
         self.assertTrue(self.run_locally_split('mynamespace.A --mynamespace.A-p 200 --expected 200'))
         self.assertFalse(self.run_locally_split('mynamespace.A --A-p 200 --expected 200'))
```

Comment:
It's okay. You can skip the `assertFalse` test. I think that that test verifies that `--A-p` is invalid syntax because it's either `--p` or `--mynamespace.A-p`.